cookiecutter/config.py

```diff
@@ -62,7 +62,7 @@ def merge_configs(default, overwrite):
 def get_config(config_path):
     """Retrieve the config from the specified path, returning a config dict."""
     if not os.path.exists(config_path):
-        raise ConfigDoesNotExistException
+        raise ConfigDoesNotExistException(config_path)
```

Comment:
Out of this issue and already changed.

Reply:
Yeah, it was useful for debugging during development. Can I just leave it in or does it conflict?